docs/releases/creating-a-release.md

```diff
@@ -13,6 +13,20 @@ _This document applies to beta and stable releases. Alpha releases are created n
 
 _This document applies to normal releases, off of `main`. For hotfixing a prior release on NPM, see [documentation on creating hotfixes](./hotfix-release.md)._
 
+## Timeline
+
+2 Week sprint. Release on 2nd week of sprint.
+
+First week of sprint, steps 0.1-0.4.
+
+Second week of sprint:
+
+Monday - 
+Tuesday - 
+Wednesday - 
+Thursday - 
+Friday - 
+
 ## Step 0: Preparation
 
 ### Step 0.1: Update Strings on Main
```

Comment:
> String translation can take up to 5 working days to complete. **This needs to be done the week before release.**

This is a little confusing. We send strings for translation after every PR. What this needs to be is something like:

```md
- String translation take up to 5 working days to complete. **Any PR changing strings needs to be complete 5 working days before release.**
```

Reply:
(I know not part of this PR, but just making notes as I run release this week)